examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/icon_with_shadow.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
```

Comment:
@StanislavPrime, don't we have the same widget in the `base` project?
As discussed we need to keep all widgets/componets that are not state-management specific in the base, so that we can achieve great code reusability

Reply:
Yes, we have the same widget in the base project and I made a change in the code to get it from there.